protocol/chat.go

```diff
@@ -282,6 +282,10 @@ func (c *Chat) IsActivePersonalChat() bool {
 	return c.Active && (c.OneToOne() || c.PrivateGroupChat() || c.Public()) && c.CommunityID == ""
 }
 
+func (c *Chat) IsPublicChat() bool {
```

Comment:
I'm a bit confused with `IsPublicChat()` and `Public()` 😵‍💫
What I meant is to have `ShouldBeSynced` method. But I guess you have some objections 🙂

Reply:
> What I meant is to have ShouldBeSynced method. But I guess you have some objections 🙂

My bad, I didn't understand you the first time. I didn't check the code snippet lol